tests/build/build.go

```diff
@@ -816,9 +817,11 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 			}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to fail")
 		})
 	})
-
+	// TODO check
```

Comment:
probably caused by an empty value [in this env var](https://github.com/redhat-appstudio/e2e-tests/blob/4a4da8c9cdcea6703eac19e19c4a6efc8ae8f0b1/scripts/install-appstudio.sh#L23-L27) (needs to be exported before bootstrapping a cluster)

Reply:
Right, removed the skip, thanks